sources/packages/backend/apps/db-migrations/src/sql/SFASBridgeLogs/Create-sfas-bridge-logs.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE sims.sfas_bridge_logs(
+    id SERIAL PRIMARY KEY,
+    reference_date TIMESTAMP WITH TIME ZONE NOT NULL,
+    generated_file_name VARCHAR(50) NOT NULL,
+    -- Audit columns (only created columns added as it is never expected to receive an update).
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT NOW(),
+    creator INT NULL DEFAULT NULL REFERENCES sims.users(id)
```

Comment:
Is there a value in adding these columns?
I would not mind having them removed or maybe the creator removed.

Reply:
I have these columns as the System is inserting this data. Please let me know if there is a downside of having them.

Reply:
For the `created_at ` I see a benefit. For the `creator` it will always be the `system`. I do not see a reason to keep the `creator`. Since it is a SFAS table it would not be a blocker.

Reply:
Removed `creator` is removed. And rollback tested.

![image](https://github.com/user-attachments/assets/bb23e109-fb8f-4fa7-9ca3-245f7f3c8648)